client/gefyra/cli/installation.py

```diff
@@ -48,8 +48,9 @@ def install(ctx, component, preset, apply, wait, **kwargs):
     from alive_progress import alive_bar
     from gefyra import api
 
-    if not all(kwargs.values()):
-        kwargs = {}
+    # filter out empty kwargs
+    kwargs = {k: v for k, v in kwargs.items() if v}
```

Comment:
It would be more transparent to show a warning when things are filtered out no? Otherwise users might wonder.